.github/workflows/release-create-hotfix.yml

```diff
@@ -15,17 +27,31 @@ jobs:
         with:
           app_id: ${{ secrets.GH_BOT_APP_ID }}
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          ref: develop
```

Comment:
it shouldn't be `main` here ?

Reply:
this is a sparse-checkout that only fetches the `tools/actions/composites` folder. It's so we can access the `./tools/actions/composites/generate-tag` action. I hadn't thought that it would matter whether it's `develop` or `main`

Reply:
I've just added a `name` for this step that makes it a bit clearer

Reply:
perfect ! thanks